sympy/combinatorics/homomorphisms.py

```diff
@@ -437,10 +437,19 @@ def group_isomorphism(G, H):
     First, the set of generators are mapped with the elements of `H` and we check if the mapping induces an isomorphism.
```

Comment:
This is a very long line - it should be split into smaller ones of <=72 symbols.

Reply:
Will it be a problem is it's GNU public licenced?